bot/src/main/java/com/community/tools/service/github/event/GithubEventsProcessingService.java

```diff
@@ -1,17 +1,14 @@
 package com.community.tools.service.github.event;
 
+import com.community.tools.service.AbstractEventProcessingService;
 import java.util.List;
-import lombok.AllArgsConstructor;
 import org.json.JSONObject;
 import org.springframework.stereotype.Service;
 
-@AllArgsConstructor
 @Service
-public class GithubEventsProcessingService {
+public class GithubEventsProcessingService extends AbstractEventProcessingService<JSONObject> {
 
-  private final List<GithubEventHandler> eventHandlers;
-
-  public void processEvent(JSONObject eventJson) {
-    eventHandlers.forEach(eventHandler -> eventHandler.handleEvent(eventJson));
+  public GithubEventsProcessingService(List<EventHandler<JSONObject>> eventHandlers) {
```

Comment:
I don't see a reason in these classes that implement AbstractEventProcessingService.
It would be better to have a base event class that encapsulate all the comon vars and behavior.